app/javascript/packs/order_screen.js

```diff
@@ -200,9 +202,8 @@ document.addEventListener('turbolinks:load', () => {
             }
           }).then((response) => {
             this.$set(this.users, this.users.indexOf(this.selectedUser), response.body.user);
-            this.$emit('updateusers');
 
-            if(!this.keepUserSelected) {
+            if(!this.keepUserSelected && this.orderRows.length === 0){
```

Comment:
Again, if the bartender already selected some products, don't close the user's view

Reply:
nice